### DISCUSSION TITLE
Multi Root Workspace Support for PnP

### DISCUSSION THREAD

**Discussion by samchouse** [yarnpkg/berry#2626] [Q&A] [answered]
I work in multi-root workspaces and I love Yarn 2 but I can't use it because I'm in the multi-root workspace. Eslint wants the nodePath to be defined for the entire workspace, and typescript has to be defined for the entire workspace. Any ideas?

**Comment by samchouse** [comment, marked as answer]
As far as I can tell it's not possible.

**Comment by merceyz** [comment]
This is a limitation of vscode so it's required to have them at the root level or you need to open vscode in each Project root
https://yarnpkg.com/getting-started/migration#editor-support
